source/release-notes/5.0.txt

```diff
@@ -776,6 +776,14 @@ The :dbcommand:`validate` command and :method:`db.collection.validate()`
 helper method also return a new :data:`~validate.repaired` boolean value
 that is ``true`` if the collection was repaired.
 
+Metadata Option Added for ``validate`` Command
```

Comment:
As noted on #6607, we probably shouldn't mention this on the 5.0 release notes since it's not applicable to all minor releases of 5.0.

Reply:
Understood, reopened a PR : https://github.com/mongodb/docs/pull/6069